### PR TITLE
Removed absolute path from LOCK_FILE in .gitignore

### PR DESCRIPTION
**Pull request by webdevium**
Because project folder location can be different on each development environment

### CODE REVIEW

File: src/Commands/AddCommand.php [hunk 1]
```diff
@@ -132,7 +133,7 @@ private function ignoreLockFile()
         $return = strpos($contents, $this->lockFile);
 
         if ($return === false) {
-            file_put_contents('.gitignore', $this->lockFile . PHP_EOL, FILE_APPEND);
+            file_put_contents('.gitignore', Hook::LOCK_FILE . PHP_EOL, FILE_APPEND);
             $this->debug("Added [{$this->lockFile}] to .gitignore");
```
Comment (BrainMaestro) [inline src/Commands/AddCommand.php]: can you update the log as well?

**Comment by BrainMaestro** [comment]
Thanks @webdevium. small oversight

File: src/Commands/AddCommand.php (hunk 1 above)
Comment (webdevium) [reply src/Commands/AddCommand.php]: Yeap. Sorry for my inattention :(

Comment (BrainMaestro) [reply src/Commands/AddCommand.php]: No worries

**Comment by BrainMaestro** [comment]
can you install `dev-master` and verify that it's correct? then I'll create a patch release

**Pull request merged**